### ISSUE TITLE
Video data are not available. 

### ISSUE THREAD

**Issue by akira-l** [SDOlivia/FineGym#10]

**Comment by akira-l** [comment]
I tried several ways to download the video data, but always missing serveral videos. I always get different warning when phrasing and capturing the frames of dataset. 
As in mmaction database: 
`video 1Fdwuy2V9EY has not been downloaded or the event clip 1Fdwuy2V9EY_E_001714_001725 not generated
video 1Fdwuy2V9EY has not been downloaded or the event clip 1Fdwuy2V9EY_E_001949_001983 not generated
video 8WGrvkd6ZEU has not been downloaded or the event clip 8WGrvkd6ZEU_E_003255_003262 not generated
video CkNz9ZIQmZI has not been downloaded or the event clip CkNz9ZIQmZI_E_004748_004827 not generated
video Lt0HnPxdzHw has not been downloaded or the event clip Lt0HnPxdzHw_E_002642_002665 not generated
video MDxIRDDWw1w has not been downloaded or the event clip MDxIRDDWw1w_E_000213_000221 not generated
video PaVNLvldLeI has not been downloaded or the event clip PaVNLvldLeI_E_004849_004962 not generated
video RLDjrhjikT4 has not been downloaded or the event clip RLDjrhjikT4_E_004334_004373 not generated
video WCcSXjsCVMA has not been downloaded or the event clip WCcSXjsCVMA_E_001617_001624 not generated
video bCDJ9CUBvqs has not been downloaded or the event clip bCDJ9CUBvqs_E_008147_008155 not generated
video bCDJ9CUBvqs has not been downloaded or the event clip bCDJ9CUBvqs_E_008227_008235 not generated
video mQ59PLtU4QA has not been downloaded or the event clip mQ59PLtU4QA_E_002522_002566 not generated
video mQ59PLtU4QA has not been downloaded or the event clip mQ59PLtU4QA_E_002650_002678 not generated
video om-qKQXxnH4 has not been downloaded or the event clip om-qKQXxnH4_E_000865_000903 not generated
video qBlozVQ7NOc has not been downloaded or the event clip qBlozVQ7NOc_E_005112_005218 not generated
video uDZVn-K4GE4 has not been downloaded or the event clip uDZVn-K4GE4_E_003068_003076 not generated
video upaJ16I0zmQ has not been downloaded or the event clip upaJ16I0zmQ_E_001911_001917 not generated
video wWwc2BbO598 has not been downloaded or the event clip wWwc2BbO598_E_000332_000340 not generated
video wYbfwx93VXk has not been downloaded or the event clip wYbfwx93VXk_E_001326_001333 not generated
`

I submitted the google form and did not achieve any response.
